From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Thu, 13 Oct 2016 21:08:18 +0200
Subject: add field 'revdate' to asciidoc files of documentation

For reproducibility we need to set up a date that's not depending on the
current time and timezone. We add a extra asciidoc control field
'revdate' and expand the variable behind that while building the package
with the date from the changelog file.
---
 doc/src/cvpcb/cvpcb.adoc                                       | 1 +
 doc/src/doc_writing_style_policy/doc_writing_style_policy.adoc | 1 +
 doc/src/eeschema/eeschema.adoc                                 | 1 +
 doc/src/gerbview/gerbview.adoc                                 | 1 +
 doc/src/getting_started_in_kicad/getting_started_in_kicad.adoc | 1 +
 doc/src/gui_translation_howto/gui_translation_howto.adoc       | 1 +
 doc/src/idf_exporter/idf_exporter.adoc                         | 1 +
 doc/src/kicad/kicad.adoc                                       | 1 +
 doc/src/pcbnew/pcbnew.adoc                                     | 1 +
 doc/src/pl_editor/pl_editor.adoc                               | 1 +
 doc/src/plugins/plugins.adoc                                   | 1 +
 11 files changed, 11 insertions(+)

diff --git a/doc/src/cvpcb/cvpcb.adoc b/doc/src/cvpcb/cvpcb.adoc
index 139467b..9eb9f08 100644
--- a/doc/src/cvpcb/cvpcb.adoc
+++ b/doc/src/cvpcb/cvpcb.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: article
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/doc_writing_style_policy/doc_writing_style_policy.adoc b/doc/src/doc_writing_style_policy/doc_writing_style_policy.adoc
index 9e34301..ba839e3 100644
--- a/doc/src/doc_writing_style_policy/doc_writing_style_policy.adoc
+++ b/doc/src/doc_writing_style_policy/doc_writing_style_policy.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: article
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/eeschema/eeschema.adoc b/doc/src/eeschema/eeschema.adoc
index bf37565..df32201 100644
--- a/doc/src/eeschema/eeschema.adoc
+++ b/doc/src/eeschema/eeschema.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: book
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/gerbview/gerbview.adoc b/doc/src/gerbview/gerbview.adoc
index 1be8bfe..1e974f2 100644
--- a/doc/src/gerbview/gerbview.adoc
+++ b/doc/src/gerbview/gerbview.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: article
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/getting_started_in_kicad/getting_started_in_kicad.adoc b/doc/src/getting_started_in_kicad/getting_started_in_kicad.adoc
index 98fc8b6..31a8702 100644
--- a/doc/src/getting_started_in_kicad/getting_started_in_kicad.adoc
+++ b/doc/src/getting_started_in_kicad/getting_started_in_kicad.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: book
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/gui_translation_howto/gui_translation_howto.adoc b/doc/src/gui_translation_howto/gui_translation_howto.adoc
index 2461fa1..86a90bb 100644
--- a/doc/src/gui_translation_howto/gui_translation_howto.adoc
+++ b/doc/src/gui_translation_howto/gui_translation_howto.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: article
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/idf_exporter/idf_exporter.adoc b/doc/src/idf_exporter/idf_exporter.adoc
index 57e3e9e..d99bec7 100644
--- a/doc/src/idf_exporter/idf_exporter.adoc
+++ b/doc/src/idf_exporter/idf_exporter.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: article
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/kicad/kicad.adoc b/doc/src/kicad/kicad.adoc
index 173dab8..37f78f7 100644
--- a/doc/src/kicad/kicad.adoc
+++ b/doc/src/kicad/kicad.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: book
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/pcbnew/pcbnew.adoc b/doc/src/pcbnew/pcbnew.adoc
index 3f229e6..332bc52 100644
--- a/doc/src/pcbnew/pcbnew.adoc
+++ b/doc/src/pcbnew/pcbnew.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: book
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/pl_editor/pl_editor.adoc b/doc/src/pl_editor/pl_editor.adoc
index d27a65a..2af44d6 100644
--- a/doc/src/pl_editor/pl_editor.adoc
+++ b/doc/src/pl_editor/pl_editor.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: article
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
diff --git a/doc/src/plugins/plugins.adoc b/doc/src/plugins/plugins.adoc
index da10b65..6628fe1 100644
--- a/doc/src/plugins/plugins.adoc
+++ b/doc/src/plugins/plugins.adoc
@@ -1,5 +1,6 @@
 :author: The KiCad Team
 :doctype: article
+:revdate: @REVDATE@
 :toc:
 :ascii-ids:
 
